cmake/CMakeLists.txt

```diff
@@ -3,14 +3,14 @@
 
 # Minimum CMake required
 cmake_minimum_required(VERSION 3.13)
-
 # Project
 project(onnxruntime C CXX)
 cmake_policy(SET CMP0069 NEW)
 set(CMAKE_POLICY_DEFAULT_CMP0069 NEW)
 
 include(CheckCXXCompilerFlag)
 include(CheckLanguage)
+set(CMAKE_VERBOSE_MAKEFILE ON)
```

Comment:
Why do you set it to ON?

Reply:
oops - this should not have been committed.